spec/messaging_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "messaging"
+
+describe Message do 
+subject(:message) {described_class.new}
+
+
+
+it "should send a text message" do
+
+ thank_you_message = "Thank you! Your order was placed and will be delivered before #{one_hours_time}"
```

Comment:
Metrics/LineLength: Line is too long. [102/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)